app/javascript/packages/document-capture/components/acuant-capture.jsx

```diff
@@ -11,27 +11,75 @@ import DataURLFile from '../models/data-url-file';
 /**
  * @typedef AcuantCaptureProps
  *
- * @prop {string}                        label      Label associated with file input.
- * @prop {string=}                       bannerText Optional banner text to show in file input.
- * @prop {DataURLFile=}                  value      Current value.
- * @prop {(nextValue:DataURLFile)=>void} onChange   Callback receiving next value on change.
- * @prop {string=}                       className  Optional additional class names.
+ * @prop {string}                        label                 Label associated with file input.
+ * @prop {string=}                       bannerText            Optional banner text to show in file
+ *                                                             input.
+ * @prop {DataURLFile=}                  value                 Current value.
+ * @prop {(nextValue:DataURLFile)=>void} onChange              Callback receiving next value on
+ *                                                             change.
+ * @prop {string=}                       className             Optional additional class names.
+ * @prop {number=}                       minimumGlareScore     Minimum glare score to be considered
+ *                                                             acceptable.
+ * @prop {number=}                       minimumSharpnessScore Minimum sharpness score to be
```

Comment:
How important is vertically aligning the params like this to you? IMO I tend to not, because it create diff noise like this? (Even with GH's ignore whitespace option it's not 100% perfect)

Reply:
> How important is vertically aligning the params like this to you? IMO I tend to not, because it create diff noise like this? (Even with GH's ignore whitespace option it's not 100% perfect)

I've started to switched my stance on it, to be honest. I think it reads nicer in columns, but as you mention, the ongoing maintenance is a headache, both in noisiness of diffs, and in the actual exercise of realigning columns†. I think I'd be fine to stop bothering with aligning them.

_† Aside: Per our Slack chats, one of the things I really miss about Sublime Text in transitioning to VSCode is how nice [DocBlockr](https://packagecontrol.io/packages/DocBlockr) is in taking away some of this burden, since it handles this in the <kbd>Tab</kbd> behavior very well. I've yet to find something in VSCode that works nearly as well._

Reply:
I started going down this route in the pull request, and considered postponing for a separate pull request, based on:

1. It might be good to apply this consistently, in a single pass.
2. Without syntax highlighting (GitHub plz), it becomes harder to decipher the property / parameter name from its description. Maybe we want to use the optional hyphen for readability?

>If you provide a description, you can make the JSDoc comment more readable by inserting a hyphen before the description.

https://jsdoc.app/tags-param.html

Reply:
Postponing until another PR is fine, the hyphen seems nice too but is probably more work to remember to add? Unless we have linting or some sort of tooling? I don't have strong feelings